src/core/dev/hw_queue_rx.cpp

```diff
@@ -54,13 +54,14 @@
 #define ALIGN_WR_DOWN(_num_wr_) (std::max(32, ((_num_wr_) & ~(0xf))))
 
 hw_queue_rx::hw_queue_rx(ring_simple *ring, ib_ctx_handler *ib_ctx,
-                         ibv_comp_channel *rx_comp_event_channel, uint16_t vlan)
+                         ibv_comp_channel *rx_comp_event_channel, uint16_t vlan, bool dummy)
     : m_p_ring(ring)
     , m_p_ib_ctx_handler(ib_ctx)
     , m_n_sysvar_rx_num_wr_to_post_recv(safe_mce_sys().rx_num_wr_to_post_recv)
     , m_rx_num_wr(align32pow2(safe_mce_sys().rx_num_wr))
     , m_n_sysvar_rx_prefetch_bytes_before_poll(safe_mce_sys().rx_prefetch_bytes_before_poll)
     , m_vlan(vlan)
+    , m_dummy(dummy)
```

Comment:
Would be better to make decision on dummy attribute in the constructor instead of storing it (lets say pass the information about dummy or number of buffers to `confiure_rq()`). If the RQ is really dummy, it won't be used and won't trigger any methods to post new buffers to the RQ.

If this cannot be done in context of constructor, then leave it in current state.